src/components/DocumentationTopic/HierarchyItem.vue

```diff
@@ -14,7 +14,7 @@
     :class="[{ collapsed: props.isCollapsed }, data.staticClass]"
     class="hierarchy-item"
   >
-    <span class="hierarchy-item-icon icon-inline">/</span>
+    <span v-if="props.hasSlash" class="hierarchy-item-icon icon-inline">/</span>
```

Comment:
Could we just make this a slot instead of introducing a special `hasSlash` prop?

Reply:
I ended up deleting the prop and using a CSS rule for it: https://github.com/apple/swift-docc-render/pull/772/commits/f189ed8d499be019d286d5c4f58069027790072a

I also took time to refactor and improve UI details of the component.